src/main/resources/avro/metadata.avdl

```diff
@@ -122,10 +122,25 @@ For data accessors, they're a simple way to scope exploration and analysis
 what's the distribution of that result in the data from our project?)   
 */
 record Dataset {
-  /** The dataset UUID. This is globally unique. */
+  /**
+  The dataset's id. This is the local/legacy/generic id.
+  */
   string id;
 
-  /** A description of the dataset. */
+  /**
+  The dataset's guid. This is globally unique id (UUIDv4).
+  TODO: Required?!
+  */
```

Comment:
@mbaudis, we need to have some API-wide design work done before
we defining any new features like GUIDs.  GA4GH doesn't even
document the meaning of `id' and `name', which has lead to
a lot of confusion.

For now, pull request should be the absolute minimum required to
get the baseline functionality working.

Michael Baudis <notifications@github.com> writes:
> In src/main/resources/avro/metadata.avdl:
> 
> >    string id;
> >
> > -  /** A description of the dataset. */
> > +  /**
> > +  The dataset's guid. This is globally unique id (UUIDv4).
> > +  TODO: Required?!
> > +  */
> 
> @dglazer @diekhans Please see #397 (and the metadata branch).
> 
> —
> Reply to this email directly or view it on GitHub.*
>